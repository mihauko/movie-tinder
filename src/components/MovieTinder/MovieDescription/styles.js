import styled from 'styled-components';


export const Root = styled.p`
    padding: 1rem;
`;
