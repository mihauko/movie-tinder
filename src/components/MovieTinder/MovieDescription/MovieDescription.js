import React from 'react';
import PropTypes from 'prop-types';

import AppContext from '../context';
import { Root } from './styles';


const MovieDescription = ({ summary }) => {
  return (
    <Root>
      {summary}
    </Root>
  );
};

MovieDescription.propTypes = {

};

export default MovieDescription;