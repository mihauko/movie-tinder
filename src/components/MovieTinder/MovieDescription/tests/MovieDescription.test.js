import { render } from '@testing-library/react';
import React from 'react';

import MovieDescription from '../MovieDescription';


describe('<MovieDescription>', () => {
  it('render component correctly', () => {
    const container = render(<MovieDescription summary="Description text" />);
    expect(container.container).toMatchSnapshot();
  });
});
