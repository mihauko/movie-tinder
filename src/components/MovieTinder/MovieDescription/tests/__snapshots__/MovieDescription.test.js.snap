// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieDescription> render component correctly 1`] = `
<div>
  <p
    class="sc-AxjAm kaYqpa"
  >
    Description text
  </p>
</div>
`;
